app/lib/theme/widgets/DecimalWidget.tsx

```diff
@@ -29,7 +39,7 @@ export const MoneyWidget: React.FC<WidgetProps> = ({
           ) {
```

Comment:
`PercentageWidget` additionally had a check to make sure the number was between 0-100, but I think there could be cases (e.g. emissions calculations) where a user might need to enter a value outside of 0-100, so I took it out.